### PR TITLE
Ecm 93

### PR DESCRIPTION
**Pull request by gauravchawla73463**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by hsjhita — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import uk.gov.hmcts.ecm.common.model.ccd.CaseData;
 import uk.gov.hmcts.ecm.common.model.ccd.types.DateListedType;
 import uk.gov.hmcts.ecm.common.model.ccd.types.HearingType;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.*;
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ListingHelper.java:9]: Doesn't CheckStyle complain about this?

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import org.springframework.util.CollectionUtils;
 import uk.gov.hmcts.ecm.common.helpers.UtilHelper;
 import uk.gov.hmcts.ecm.common.idam.models.UserDetails;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.*;
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java:9]: CheckStyle again

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/timetofirsthearing/TimeToFirstHearingReportTest.java [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        ListingDetails listingDetails = new ListingDetails();
+        listingDetails.setCaseTypeId(NEWCASTLE_LISTING_CASE_TYPE_ID);
+        ListingData caseData = new ListingData();
+        listingDetails.setCaseData(caseData);
+        List<SubmitEvent> submitEvents = new ArrayList<>();
+        TimeToFirstHearingReport timeToFirstHearingReport = new TimeToFirstHearingReport();
+        ListingData listingData = timeToFirstHearingReport.generateReportData(listingDetails, submitEvents);
+        verifyReportHeaderIsZero(listingData);
+    }
+    private void verifyReportHeaderIsZero(ListingData listingData) {
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/timetofirsthearing/TimeToFirstHearingReportTest.java:33]: Missing a new line

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/timetofirsthearing/TimeToFirstHearingReportTest.java [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+        ListingData reportListingData = timeToFirstHearingReport.generateReportData(listingDetails, submitEvents);
+
+        AdhocReportType adhocReportType = reportListingData.getLocalReportsDetailHdr();
+        assertEquals(1, Integer.parseInt(adhocReportType.getTotalCases()));
+        assertEquals(0, Integer.parseInt(adhocReportType.getTotal26wk()));
+        assertEquals(1, Integer.parseInt(adhocReportType.getTotalx26wk()));
+        assertEquals(0, Float.parseFloat(adhocReportType.getTotal26wkPerCent()), .00);
+        assertEquals(100, Float.parseFloat(adhocReportType.getTotalx26wkPerCent()), .00);
+    }
+
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/timetofirsthearing/TimeToFirstHearingReportTest.java:130]: Too many new lines

**Review by jthmcts — APPROVED** [review]

**Pull request merged**